commit 4d2d6395e5ec033dffc0678ba1d3ab6de05a1514
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2019-07-05T18:52:47+02:00 (3 months ago)
    
    TRITON-1783 VM creation for bhyve and kvm VMs shouldn't pass `quota` attribute to `vmadm`. Just `disks` and `flexible_disk_size`

diff --git a/lib/workflows/provision.js b/lib/workflows/provision.js
index 8a53444..3125346 100644
--- a/lib/workflows/provision.js
+++ b/lib/workflows/provision.js
@@ -5,11 +5,12 @@
  */
 
 /*
- * Copyright (c) 2019, Joyent, Inc.
+ * Copyright 2019 Joyent, Inc.
  */
 
 /*
- * A brief overview of this source file: what is its purpose.
+ * This is the VM provision workflow, used to create provision jobs
+ * for Triton VMs.
  */
 
 var async = require('async');
@@ -23,7 +24,7 @@ var nfsVolumes = require('./nfs-volumes');
 
 var wfapiUrl;
 
-var VERSION = '8.2.1';
+var VERSION = '8.2.2';
 
 
 /*
@@ -231,6 +232,10 @@ function preparePayload(job, cb) {
                 payload[field] = job.params.image[field];
             }
         });
+
+        // Rely into default vmadm values with `disks` and `flexible_disk_size`
+        // for KVM/Bhyve VMs:
+        delete payload.quota;
     } else {
         payload['image_uuid'] = params['image_uuid'];
 
